test/test_runner.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
This is a small version of what a test runner that uses `metamocha` would look like. It would operate very similarly to `mocha`, where you create an instance of the runner, add the files, and then run it, but `Metamocha` also exposes methods to add filters and configuration.